designs/virtualization/4.5/Scheduling/Firmware-config/Firmware-config.md

```diff
@@ -0,0 +1,32 @@
+---
+parent: virtualization
+version: 4.5
+---
+
+# Firmware Config
+
+Exposing Firmware Config for VM.
+Firmware is placed within the scheduling section. It is related to CPU, and shoudn't be seperated from the rest of the CPU features. Because of that, it should be located next to other CPU features. Also, it affects performance, so it is related to "Resource Requirements".
+
+![VM page](img/Firmware-0-0.jpg)
+Firmware Config is placed in the Scheduling sction of the VM details tab.
+The default value of this field is "Default"
+
+![Config modal](img/Firmware-1-0.jpg)
```

Comment:
@Ranelim This is minor but just wondering if this should really be called Firmware Configuration rather than "Config" for consistency. That seems to be what we have done elsewhere in the console such as the Global config area is called Global Configuration for the actual tab.

Reply:
After talking with the dev team we dropped the Config so the field name will be named 'Firmware'. This is because essentially everything is "configuration" and this field applies specifically for the firmware. Does that work?